My posted job

zengni2018 opened the pull request:
created employer published jobs page;
can see the posted jobs,also can choose jobs by id.

zengni2018 committed:
mypostedjob

create a mypostedjob php file

zengni2018 committed:
Create employer_published_jobs.php

zengni2018 committed:
Merge branch 'master' into my-posted-job

zengni2018 committed:
Update employer_published_jobs.php

AxelBogos commented on .gitignore:
```diff
@@ -1,3 +1,4 @@
 *.xml
 config.php
 
+.idea/dataSources/952722df-8d9a-420e-b7c2-b7305554fb8f/storage_v2/_src_/schema/information_schema.FNRwLQ.meta
```
Please remove this file from the commit

zengni2018 replied:
done

AxelBogos commented on mypostedjob.php:
```diff
@@ -0,0 +1 @@
+<?php
```
Please remove this file from the commit

zengni2018 replied:
done

AxelBogos commented on employer_published_jobs.php:
```diff
@@ -0,0 +1,100 @@
+</HEAD>
+
+<BODY>
+<?php require 'user_dashboard_navbar.php' //nav bar
```
Change to employer_dashboard_navbar

zengni2018 replied:
done

AxelBogos commented on employer_published_jobs.php:
```diff
@@ -0,0 +1,100 @@
+    </div>
+</td>
+</tr>
+<tr>
+    <td></td>
+    <td style="text-align: center">
+        <div id="longJobDescription"></div>
+        <form id='jobAppliedForm' action="<?php echo htmlspecialchars($_SERVER["PHP_SELF"]); ?>" style="visibility: hidden" method="post">
+            <div class="form-group">
+                <input type="text" style="visibility: hidden" id= 'appliedJob' name="appliedJob" value="">
+                <br>
+                <input type="submit" class="btn btn-primary" value="Apply" align="left">
+            </div>
+
+        </form>
+    </td>
+</tr>
```
Remove this table row for now

zengni2018 replied:
done

zengni2018 committed:
my posted job

remove .gitignore;
remove mypostedjob.php;
Change to employer_dashboard_navbar;
Remove a table row for now
